### DISCUSSION TITLE
There are still leftover files when running make clean

### DISCUSSION THREAD

**Discussion by ghost** [premake/premake-core#2350] [General]
Is it an issue of premake or that particular project? And why does premake choose the .lib extension instead of .a when the toolchain is not MSVC but GCC/Clang?

Please have a look:

https://github.com/SpartanJ/eepp/issues/81

**Comment by nickclark2016** [comment]
Please convert this to an issue. I would agree it's a deficiency on the gmake2 generator.

**Reply by ghost** [reply]
I will do as you said.
